plugin/src/main/java/org/opensearch/sql/plugin/config/OpenSearchPluginModule.java

```diff
@@ -108,6 +108,6 @@ public QueryPlanFactory queryPlanFactory(DataSourceService dataSourceService,
     Planner planner = new Planner(LogicalPlanOptimizer.create());
     QueryService queryService = new QueryService(
         analyzer, executionEngine, planner);
-    return new QueryPlanFactory(queryService, planSerializer);
+    return new QueryPlanFactory(queryService);
```

Comment:
`PlanSerializer` is not needed here anymore as long as a function which `@Provides` it